types/markdown-it-lazy-headers/markdown-it-lazy-headers-tests.ts

```diff
@@ -0,0 +1,13 @@
+import * as MarkdownIt from "markdown-it";
```

Comment:
I don't believe it's correct to use `new` on an ES6 module.  Maybe this ought to be a Node-style import?

Reply:
It's true that it compiles at is, but it's taking advantage of a gap in the compilers checks (i.e. working by accident).  If you used an import like `import MarkdownIt = require("markdown-it");`, it would work by design.

Reply:
Do you have any pointer WHY that is in this case? 
I modified it anyways.. but don't fully understand yet..

Reply:
I found this explanation helpful, if a little snarky: https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this